src/App.tsx

```diff
@@ -1,93 +1,17 @@
-/* eslint-disable jsx-a11y/control-has-associated-label */
 import React from 'react';
+import { TodoApp } from './components/todo_app/TodoApp';
+import { TodoList } from './components/todo_lilst/TodoList';
```

Comment:
```suggestion
import { TodoList } from './components/TodoList/TodoList';
```
Typo in the path. Also, it's better to use PascalCase for folders' names